src/shared/ui/common/variablesPrompter.ts

```diff
@@ -23,7 +23,7 @@ function unquote(str: string): string {
     const isDoubleQuoted = str[0] === '"' && str[str.length - 1] === '"'
 
     if (isSingleQuoted || isDoubleQuoted) {
-        str = str.substr(1, str.length - 1)
+        str = str.slice(1, -1)
```

Comment:
This looks like a logic change:

```
> 'abcd'.substr(1, 'abcd'.length - 1)
'bcd'
> 'abcd'.slice(1, -1)
'bc'
```

Reply:
oh, it actually fixes a bug :) It's a good idea to mention that in the commit message.

```
> str = '"abcd"'
'"abcd"'
> str.substr(1, str.length - 1)
'abcd"'
> str.slice(1, -1)
'abcd'

Reply:
You're right. Looking at the check above which checks the first and last char I thought this was the inteded behaviour.
As you stated though it's different to the current behaviour which only cuts of the first char.

Reply:
Yup it was a bug. I thought I had fixed it a while ago but the change (plus test) must've never gotten merged.